config/storybook/styles/_storybook-global.scss

```diff
@@ -56,3 +56,13 @@ h6 {
     transform: translate3d(0, 64px, 0);
   }
 }
+
+// Grid
+.st-grid-cell-visualiser {
```

Comment:
👍

Reply:
Likely we'll have to split this file in to more partials as Storybook grows, `st-` seemed like a good starting point for namespacing 🤘🏻

Reply:
perhaps, all good for now tho